microservices/rosettafold/microservice/api.py

```diff
@@ -0,0 +1,40 @@
+import pathlib
+from typing import Optional, Annotated
+
+from fastapi import FastAPI, UploadFile, File
+
+from microservice.api_models import RunRosettaFoldRequest, RunRosettaFoldResponse
+from microservice.services import RosettaService
+
+app = FastAPI(
+    title='RoseTTAFold API',
+    version='0.1'
+)
+
+from microservice.loggers import logger
+
+
```

Comment:
We need to add:

```suggestion
@app.get("/job/{job_id}/is-running")
def is_job_running(job_id: str) -> IsJobRunningResponse:
    return IsJobRunningResponse(is_running=job_state_manager.is_job_running(job_id))

@app.get("/jobs/running")
def get_running_jobs():
    return {"running_jobs": job_state_manager.get_running_jobs()}

```

For docking part to keep track of what jobs are running currently (also will be used by other services). See `esmfold` microservice for the example